### DISCUSSION TITLE
WARNING message about PATH being altered to include directory when not needed using pipx

### DISCUSSION THREAD

**Discussion by Nani-o** [ansible/ansible-lint#3259] [General]
As stated in Issue #2048, there is an incorrect warning message when using pipx to install ansible-lint.

> @nervo That has nothing to do with symlinks. I will lock this issue to prevent further noise. Feel free to use discussions if help is needed.

It has, when using pipx there is venvs for each package you install, and a bin folder `~/.local/pipx/venvs/PACKAGE/bin`, theses binaries are in turn symlinked in `~/.local/bin`.

It makes sense, when using pipx to install packages, to only have `~/.local/bin` in your PATH, and not to include every venv bin folder to your PATH.

If you want to reproduce :

```
$ pipx install ansible-lint
[...]
$ export PATH=$PATH:~/.local/bin
$ ansible-lint
WARNING: PATH altered to include /home/redacted/.local/pipx/venvs/ansible-lint/bin :: This is usually a sign of broken local setup, which can cause unexpected behaviors.

Passed with production profile: 0 failure(s), 0 warning(s) on 0 files.
```

**Comment by Nyxtorm** [comment]
Same here, my ansible-lint is a symlink in /usr/local/bin/ (in $PATH) and I have the warning.
